### DISCUSSION TITLE
Migrating from old nzbget?

### DISCUSSION THREAD

**Discussion by TylerHomeLab** [nzbgetcom/nzbget#100] [Q&A] [answered]
I have an original verzion nzbget install on windows that stopped updating as the project died.  

Do I just install over it?  Anything I should be concerned about?

**Comment by phnzb** [comment, marked as answer]
Migration from old nzbget:

1. All platforms / Generic instructions:
- backup all nzbget settings (Settings - System - Backup Settings)
- stop all running nzbget instances
- uninstall old application
- remove old config file (strongly recommended)
- install new version
- restore needed settings (Settings - System - Restore Settings)

3. Platform-specific:

Windows:
- config file location: C:\ProgramData\NZBGet\nzbget.conf

Linux (installed via .run package):
- config file location: {AppDir}/nzbget.conf
- backup old config file (from WebUI or manually). If `downloads` dir is in nzbget directory, backup it as well (or move temporarily to another location).
- remove old nzbget directory with all contents
- install new nzbget version
- restore old config file, old `downloads` (if needed)

macOS:
- config file location: ~/Library/Application Support/NZBGet/nzbget.conf 
- just remove old NZBGet.app from Applications and copy new NZBGet.app (settings would be saved, config file not removed during this process)

Synology (migrate from SynoCommunity package):
- backup settings via WebUI
- remove old package completely (with wipe settings)
- install spk for your platform
- restore needed settings
- please note - if MainDir changed during restore settings, this may require editing permissions for the `nzbget` user. More info - in Synology readme.

Docker:
- Check complete [guide](https://github.com/nzbgetcom/nzbget/issues/84#issuecomment-1884846500)

Known issues:
- if upgrading from nzbget-ng to nzbget-stable (22.0), or downgrading from testing build to stable - an error may occur with the `ArticleReadChunkSize` setting. Please remove it manually from config file, this setting works only with testing builds now

**Reply by TylerHomeLab** [reply]
Thank you!  That worked as expected.  Much appreciated.

**Comment by demonbane** [comment]
If you were previously using the Ubuntu packaged version of nzbget 21 you're going to have some system paths included in the config backup that can cause a few issues. Doing a full uninstall of the package first **might** fix it (I'm not sure what happens when the path specified doesn't exist), but you can also just edit the backed up config file and update the `WebDir=` and `ConfigTemplate=` lines to point to the new install manually.
